verification.py

```diff
@@ -226,6 +227,20 @@ def make_test_dir(test_name):
     return basedir
 
 
+class Test:
```

Comment:
Let's start using [attrs](https://www.attrs.org/en/stable/) for these classes. There's examples of using this in, e.g., the demography.py module.